flutter/lib/src/screenshot/sentry_screenshot_quality.dart

```diff
@@ -0,0 +1,19 @@
+enum SentryScreenshotQuality {
+  full,
+  high,
+  medium,
+  low;
+
+  int? targetResolution() {
+    switch (this) {
+      case SentryScreenshotQuality.full:
+        return null; // Keep current scale
+      case SentryScreenshotQuality.high:
+        return 1920;
+      case SentryScreenshotQuality.medium:
+        return 1280;
+      case SentryScreenshotQuality.low:
+        return 854;
```

Comment:
Did you come up with the numbers from somewhere?

Reply:
NVM asked @bitsandfoxes since its the same on Unity.

Reply:
Yeah i took the values from there, assuming they are sane defaults, as they are already in use.d